### DISCUSSION TITLE
Bot -- Mentioning user without its ID

### DISCUSSION THREAD

**Discussion by Simon-Busch** [discordjs/discord.js#8152] [Q&A] [answered]
Hey community !

I would like to have your light here, I am building a bot for our project and would like to tag user occasionally but don't have access to the user ID.

My question: 
Is it possible to tag the user with only their user handle [ex: test#1111] ?

I tried a few approaches with `<HANDLE>`  `<@handle>` `<@!handle>`

Your help would be highly appreciated 🙏🏼

**Comment by Jiralite** [comment, marked as answer]
It is not possible to mention someone without their id.

If you have their handle, the most you can do is perform a fetch or search in the server (provided they are in it) to check for a match and use the matching result, if any, to retrieve their id and mention them.

**Reply by Simon-Busch** [reply]
@Jiralite Blazing fast ! Thank you for your prompt reply ! 

Is it something planned in the future development for discord.js ?

**Reply by Jiralite** [reply]
This isn't a discord.js limitation. It's simply not possible.

**Reply by Simon-Busch** [reply]
Very clear, thank you
